src/controllers/csi/gc/unmounted.go

```diff
@@ -32,12 +33,12 @@ func (gc *CSIGarbageCollector) getUnmountedVolumes(tenantUUID string) ([]os.File
 	}
 
 	for _, volumeID := range volumeIDs {
-		unused, err := afero.IsEmpty(gc.fs, gc.path.OverlayMappedDir(tenantUUID, volumeID.Name()))
+		isUnused, err := afero.IsEmpty(gc.fs, gc.path.OverlayMappedDir(tenantUUID, volumeID.Name()))
```

Comment:
a little nitpick but do we want to use negated names for booleans? i'm thinking of the case !isUnused, in my opinion isUsed would be more readable. but as I mentioned only a thought.